stan/math/prim/meta/size_mvt.hpp

```diff
@@ -9,29 +9,29 @@
 namespace stan {
```

Comment:
If you are really sure they should live inside `stan::math`, then this needs to be fixed.